### PR TITLE
XWIKI-18579: Inconsistent rounding of save buttons at the bottom of the edit forms (in-place or not)

### PR DESCRIPTION
**Pull request by manuelleduc**
Jira: https://jira.xwiki.org/browse/XWIKI-18579

### CODE REVIEW

File: xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/buttons.less [hunk 1]
```diff
@@ -183,4 +183,8 @@ input.button, .buttonwrapper button, .buttonwrapper a {
     color: @navbar-default-link-hover-color;
     background-color: @navbar-default-link-hover-bg;
   }
+}
+
+.btn-group > .btn:not(:first-child):not(:last-child):not(.dropdown-toggle) {
+  border-radius: 0 @border-radius-base @border-radius-base 0;
```
Comment (manuelleduc) [inline xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/buttons.less:189]: To show the general idea, I'll update with a comment before removing the draft status.

Comment (manuelleduc) [reply xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/buttons.less:189]: Actually, not a good selector, it was only working on a specific case but is not generic enough

File: xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editactions.vm [hunk 2]
```diff
@@ -29,8 +29,18 @@
   <div class="buttons">
     <div class="btn-group">
       #editActionButton($saveaction, 'saveandview', 'btn-primary')
-      #editActionButton('saveandcontinue', 'save', 'btn-default')
-      #if ($previewenabled) #editActionButton('preview', 'preview', 'btn-default') #end
+      #set ($saveAndContinueClasses = 'btn-default')
+      #set ($previewClasses = 'btn-default')
+      ## Add a btn-group-last class to the last visible elements of the button group as the hidden fields 
+      ## added by editActionButton are ignored by the css selectors, preventing them from having roundered
+      ## corners on their right side.s
```
Comment (mflorea) [inline xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editactions.vm:36]:
```suggestion
      ## corners on their right side.
```

File: xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editactions.vm [hunk 3]
```diff
@@ -29,8 +29,18 @@
...
+      #if($previewenabled)
```
Comment (mflorea) [inline xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editactions.vm:37]:
```suggestion
      #if ($previewenabled)
```

**Review by mflorea — APPROVED** [review]
It feels like a hack / workaround, but unfortunately I don't have a better solution. Wrapping each button and its corresponding hidden input in a ``span.btn`` is more risky in terms on backward compatibility and may require more CSS styling. So let's go with this solution.

**Pull request merged**